week-2-practise/level-1/2-validation.js

```diff
@@ -2,7 +2,7 @@
 // TIP: You should write an expression that returns a boolean value
 
 var num = 10;
-var isBigEnough; // ONLY EDIT THIS LINE
+var isBigEnough = true; // ONLY EDIT THIS LINE
```

Comment:
I don't think this is quite the correct answer. The exercise is asking you to calculate if `num` is bigger or equal to 10. What happens if we change the `num` variable to 12?